manager/testdata/new-blueprints/scenario5/plotter.yaml

```diff
@@ -0,0 +1,91 @@
+# This example has no input source for simplification
+# Let's assume it's outputting a random test dataset of processed paysim data.
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: Plotter
+metadata:
+  name: app1-ns1 # plotter name is a combination of app and namespace
+  namespace: m4d-system
+  labels:
+spec:
+  appSelector: # Selector of the application that uses this workload
+    clusterName: thegreendragon
+    workloadSelector:
+      matchLabels:
+        app: demoapp
+  assets:
+  - dataSetID: "paysim-processed"  # dataset requested by user
+    source:
+      credentials:
+        write:
+          vault:
+            address: http://vault.m4d-system:8200
+            authPath: /v1/auth/kubernetes/login
+            role: module
+            secretPath: /v1/kubernetes-secrets/paysim?namespace=m4d-notebook-sample
+      connection:
+        s3:
+          endpoint: localhost:8001
+          bucket: srcbucket
+          object: data.csv
+      dataformat: csv
+  flows:
+  - name: flow-paysim
+    flowType: write
+    dataSetID: "paysim-processed"
+    subflows:
+    - name: write
+      flowType: write
+      trigger: workload
+      steps:
+      - - name: step1
+          cluster: thegreendragon
+          template: writeTemplate
+          parameters:
+            sink: # This technically has multiple sinks. One being the dataset and the other the following step2
```

Comment:
So should sink be a list? It would have to support different types of sinks, since here the first is a file and the second is an API?